### ISSUE TITLE
Info on usage outside

### ISSUE THREAD

**Issue by docop** [Ahmed-jerbi/VIOSO-Unity-HDRP#2]
Hi
 your setup is quite great and nice find of the blend function.. I wonder, was it only possible to do blend via unity... or other software or just having a desktop can be working ?  Like have quadro card, but never manage to be able to do any blend or so...

thanks in advance

**Comment by AJvioso** [comment]
Yes we have a plugin for other major rendering applications and also a native integration into nvidia API via quadro/RTX pro cards for windows desktop warping and blending. Here's the software page with the full feature list: [https://vioso.com/software/vioso6/](https://vioso.com/software/vioso6/)
